packages/react-scripts/template/src/components/example/LearnReactCard.js

```diff
@@ -1,6 +1,6 @@
 import React from 'react'
 import { Button, Card, CardMedia, CardActions, CardContent } from '@fs/zion-ui'
-import ReactImage from './reactjs.jpg'
+import ReactImage from './reactjs.webp'
```

Comment:
webp won't work on Safari. Should we do a responsive/adaptive image, or just switch to png/jpg again

Reply:
Good point, I'll rollback to jpgs

Reply:
I would like to have a pattern/component that serves webp to browsers that support it and fallsback to jps for the rest. Using the picture element and other techniques at some point.